server/routes/service.py

```
@@ -0,0 +1,559 @@
+
+@services_bp.route("/services/<int:id>", methods=["PUT"])
+@jwt_required()
+def update_service(id):
+    """
+    Update a service. Only for admin and super_admin users.
+    Image is automatically converted to base64 in to_dict() method.
+    """
+    try:
+        service = Service.query.get(id)
+        if not service:
+            raise NotFound(f"Service with ID {id} not found")
```

Ruff (0.13.3)

Abstract `raise` to an inner function

(TRY301)